pkg/controller/core/workload_controller.go

```diff
@@ -607,9 +607,8 @@ func (r *WorkloadReconciler) Create(e event.CreateEvent) bool {
 	workload.AdjustResources(ctx, r.client, wlCopy)
 
 	if !workload.HasQuotaReservation(wl) {
-		err := r.queues.AddOrUpdateWorkload(wlCopy)
-		if err != nil {
-			log.V(2).Info(fmt.Sprintf("%s; ignored for now", err))
+		if err := r.queues.AddOrUpdateWorkload(wlCopy); err != nil {
+			log.V(2).Info("ignored an error for now", "error", err)
```

Comment:
It would be better to use structured logging instead of wrapping values inside the message.